st2client/st2client/commands/triggerinstance.py

```diff
@@ -117,14 +118,24 @@ def run(self, args, **kwargs):
         if args.status:
             kwargs['status'] = args.status
 
-        return self.manager.query(limit=args.last, **kwargs)
+        result, count = self.manager.query(limit=args.last, **kwargs)
+        return (result, count)
 
     def run_and_print(self, args, **kwargs):
-        instances = self.run(args, **kwargs)
-        self.print_output(reversed(instances), table.MultiColumnTable,
-                          attributes=args.attr, widths=args.width,
-                          json=args.json, yaml=args.yaml,
-                          attribute_transform_functions=self.attribute_transform_functions)
+
+        instances, count = self.run(args, **kwargs)
+        if args.json or args.yaml:
+            self.print_output(reversed(instances), table.MultiColumnTable,
+                              attributes=args.attr, widths=args.width,
+                              json=args.json, yaml=args.yaml,
+                              attribute_transform_functions=self.attribute_transform_functions)
+        else:
+            self.print_output(reversed(instances), table.MultiColumnTable,
+                              attributes=args.attr, widths=args.width,
+                              attribute_transform_functions=self.attribute_transform_functions)
+            if args.last >= self.default_limit and count and int(count) > args.last:
```

Comment:
Hmm, it is working for me. If n > default limit then show the note, otherwise, just show the result.
```
$ st2 rule-enforcement list -n 1
+--------------------------+----------------------+----------------------+----------------------+----------------------+
| id                       | rule.ref             | trigger_instance_id  | execution_id         | enforced_at          |
+--------------------------+----------------------+----------------------+----------------------+----------------------+
| 5952ab31909a5027b65c037b | st2ci.st2_pkg_test_a | 5952ab30909a5027b65c | 5952ab30909a5027b65c | 2017-06-27T19:00:00. |
|                          | nd_promote_unstable_ | 036e                 | 037a                 | 858196Z              |
|                          | u16                  |                      |                      |                      |
+--------------------------+----------------------+----------------------+----------------------+----------------------+

$ st2 execution list -n 1
+--------------------------+------------+--------------+------------------------+------------------+---------------+
| id                       | action.ref | context.user | status                 | start_timestamp  | end_timestamp |
+--------------------------+------------+--------------+------------------------+------------------+---------------+
| 595467b3909a5015dab1c8b9 | core.local | stanley      | succeeded (1s elapsed) | Thu, 29 Jun 2017 | Thu, 29 Jun   |
|                          |            |              |                        | 02:36:35 UTC     | 2017 02:36:36 |
|                          |            |              |                        |                  | UTC           |
+--------------------------+------------+--------------+------------------------+------------------+---------------+

$ st2 trigger-instance list -n 1
+--------------------------+---------------------------+---------------------------+-----------+
| id                       | trigger                   | occurrence_time           | status    |
+--------------------------+---------------------------+---------------------------+-----------+
| 59546ab5909a5045fc0d0882 | core.st2.key_value_pair.c | Thu, 29 Jun 2017 02:49:25 | processed |
|                          | reate                     | UTC                       |           |
+--------------------------+---------------------------+---------------------------+-----------+
```